.eslintrc.js

```diff
@@ -4,9 +4,7 @@ module.exports = {
     jest: true
   },
   rules: {
-    'no-proto': 0,
-    strict: 0,
-    'max-len': 0,
```

Comment:
The rule 'max-len' has been removed. This could potentially lead to lines of code that are too long and decrease readability.